scripts/make_nodes

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
where do u use this?

Reply:
You can't change the /etc/hosts file while building the docker image - you have to do from within the container - so i just made a script to add the hosts to the /etc/host file for me.

Reply:
I took Steven's advice below and added the nodes in the docker-compose yaml file via the extra_hosts property.